recipes/linalg/25.10/amdgpu/config.yaml

```
@@ -0,0 +1,11 @@
+name: linalg
+spack:
+  repo: https://github.com/spack/spack.git
+  commit: releases/v1.0
```

```suggestion
  commit: releases/v1.1
```
Wort trying the new version?